_layouts/default.html

```diff
@@ -89,6 +89,12 @@
     </div>
     <div class="flex max-width">
         <div class="defaultwrapper" id="defaultwrapper">
+        <input type="checkbox" class="openSidebarMenu" id="openSidebarMenu" />
+        <label for="openSidebarMenu" class="sidebarIconToggle">
```

Comment:
For accessibility, this should have some (hidden) text, and perhaps even some `aria` role.